home_location:
- -0.032391386121858635
- 0.9082940152928102
- 0.0
- 0.0
- 0.0
- -0.9703032903566129
- 0.24189155570860774
make_location:
- -0.03434295117769088
- 0.1947483679449661
- 0.0
- 0.0
- 0.0
- -0.8329617610653425
- 0.5533305563611352
table1_location:
- -0.2657934333309922
- -1.0890767799803263
- 0.0
- 0.0
- 0.0
- -0.7628855300954481
- 0.6465335783785611
table2_location:
- -1.7075500914592079
- -1.4567949923854961
- 0.0
- 0.0
- 0.0
- -0.9423541489570784
- 0.334617181183783
table3_location:
- -2.5477274418144926
- 0.48291808973727457
- 0.0
- 0.0
- 0.0
- -0.9843290549187594
- 0.17634146319780203
